PayWithMyBankSDKDemoAndroid/app/src/main/java/net/trustly/paywithmybanksdkdemoandroid/EstablishData.kt

```diff
@@ -0,0 +1,28 @@
+package net.trustly.paywithmybanksdkdemoandroid
+
+object EstablishData {
+
+    fun getEstablishDataValues(): Map<String, String> {
+        val establishDataValues: MutableMap<String, String> = HashMap()
+        establishDataValues["accessId"] = "A48B73F694C4C8EE6307"
+        establishDataValues["requestSignature"] = "clYlSWb565B3JGUExdXrK5tyTqM="
+        establishDataValues["merchantId"] = "1009542823"
```

Comment:
Please change all IDs to a string constant and then leave the strings as "YOUR_MERCHANT_ID" or "YOUR_ACCESS_ID" etc.